docs/options.md

```diff
@@ -124,12 +124,12 @@ CIBW_SKIP: cp27-* cp35-*
 # Skip Python 3.6 on Linux
 CIBW_SKIP: cp36-manylinux*
 
-# Only build on Python 3 and skip 32-bit builds
-CIBW_BUILD: cp3?-*
+# Only build on Python 3 (ready for 3.10 when it comes) and skip 32-bit builds
+CIBW_BUILD: cp3{?,??}-*
```

Comment:
This could just be `3*`, then, or is that nothing we want to advise?

Reply:
We do this more or less already with `pp*`, so don't think the `-` is important. But I wanted somewhere to show the brackets. ;)

Reply:
Ah, yes `cp3*-` might be a neater way to write it. If we wanted to demo the brackets, we could do `{cp,pp}3*-`? (also this matches the comment)

Reply:
Good call, @joerick :-)